### PR TITLE
Prune changelog for beta.6

### PR DESCRIPTION
**Pull request by prprabhu-ms**
Prune the changelog down for humans.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+Tue, 28 Sep 2021 19:19:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@internal/react-components_v1.0.0-beta.5..@internal/react-components_v1.0.0-beta.6)
+
+### Changes
+
+* Support editing and deleting sent messages in the `MessageThread` UI
+* Support customization to the `ParticipantList` to inject menu items for each participant
+* (breaking change) API consistency and documentation updates
+  * Add a uniform way to set cutomization options on the composites.
+* Add `MeetingAdapter` to back the `MeetingComposite`. By default, the `MeetingAdapter` is backed by the Azure Communication Services backends.
```
Comment (prprabhu-ms) [inline packages/communication-react/CHANGELOG.md:18]: @JamesBurnside Do we want to mention `MeetingComposite` in the release notes at this point?

File: packages/communication-react/CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
 
-This log was last generated on Mon, 13 Sep 2021 21:02:16 GMT and should not be manually modified.
+This log was last generated on Tue, 28 Sep 2021 19:19:18 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.0.0-beta.6](https://github.com/azure/communication-ui-library/tree/@internal/react-components_v1.0.0-beta.6)
+
+Tue, 28 Sep 2021 19:19:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@internal/react-components_v1.0.0-beta.5..@internal/react-components_v1.0.0-beta.6)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:10]: Something for James to fix in the future, this should be something like: https://github.com/azure/communication-ui-library/compare/1.0.0-beta.5..1.0.0-beta.6

File: packages/communication-react/CHANGELOG.md (hunk 1 above)
Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:18]: Yeah we should be fine too - would think about adding some sort of (BETA) but probably unecessary

**Review by JamesBurnside — APPROVED** [review]

File: packages/communication-react/CHANGELOG.md (hunk 2 above)
Comment (prprabhu-ms) [reply packages/communication-react/CHANGELOG.md:10]: Actually, not sure if this is James or Prathmesh that needs a fix.

I didn't have any entries in the changelog here at all. I started by copy-pasta'ing in the changes from all the packlets. Now you know which one I copied first.

More importantly though, I think your new link won't work either: 
image

I think the problem is that we delete the release branch when we're done (so there's no https://github.com/Azure/communication-ui-library/tree/release/1.0.0-beta.5 anymore).

File: packages/communication-react/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+
+### Changes
+
+* Support editing and deleting sent messages in the `MessageThread` UI
+* Support customization to the `ParticipantList` to inject menu items for each participant
+* (breaking change) API consistency and documentation updates
+  * Add a uniform way to set cutomization options on the composites.
```
Comment (alcail) [inline packages/communication-react/CHANGELOG.md:17]: nit: extra space to be removed

File: packages/communication-react/CHANGELOG.md (hunk 2 above)
Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:10]: It smore we haven't created the beta.6 tag, its only created at release time, e.g. comparing beta-4 and beta-5 is: https://github.com/azure/communication-ui-library/compare/1.0.0-beta.4..1.0.0-beta.5

**Review by alcail — APPROVED** [review]

File: packages/communication-react/CHANGELOG.md (hunk 3 above)
Comment (prprabhu-ms) [reply packages/communication-react/CHANGELOG.md:17]: Added more spaces. Intended it to be a sub-list :)

**Pull request merged**